packages/create-playwright/src/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
note: NCC does not traverse tsconfig's up and they fallback to their default one. Thats why we need to reference it two times:
a) rootDir of create-playwright for VSC to pick it up
b) rootDir/src/ of create-playwright so NCC does pick it up